client/package-lock.json

```diff
@@ -1,15797 +0,0 @@
-{
```

Comment:
Are we deleting `package-lock.json` for real @ankiiitraj? We'll be using yarn completely?

Reply:
I didn't asked to delete client/package-lock.json but the package-lock.json in the main folder only.
Nonetheless @sandesh-08 add this file back. 
And not will not be using yarn in this project.

Reply:
Great that we got everything cleared now. So @sandesh-08, you know what to do! 😉